<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>983</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>983</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>983</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>983</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>进程名称：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_pname"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>创建时间：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_createtime"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>执行时间：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_runningtime"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="craete_pushButton">
            <property name="text">
             <string>创建</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTableWidget" name="tableWidget_ready">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>154</number>
         </attribute>
         <column>
          <property name="text">
           <string>进程名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>创建时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>执行时间</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>算法选择</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_1">
            <property name="text">
             <string>先来先服务算法</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>最短作业优先调度算法（非抢占式）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>最短作业优先调度算法（抢占式）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string>最高响应比优先调度算法</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>平均带权周转时间：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="delete_pushButton">
         <property name="text">
          <string>完成</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>平均周转时间：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <property name="columnCount">
          <number>6</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>120</number>
         </attribute>
         <column>
          <property name="text">
           <string>进程名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>进程状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>开始时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>完成时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>周转时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>带权周转时间</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
